### PR TITLE
[tests-only] add test for renaming file and folder

### PR DESCRIPTION
**Pull request by Salipa-Gurung**
### Description 
This PR adds GUI test for renaming a file and a folder
### Related Issue 
https://github.com/owncloud/QA/issues/789

### CODE REVIEW

**Review by amrita-shrestha — APPROVED** [review]
lgtm

File: test/gui/tst_moveFilesFolders/test.feature [hunk 1]
```diff
@@ -41,4 +41,15 @@ Feature: move file and folder
...
+
+    Scenario: Rename file and folder
+        Given user "Alice" has uploaded file with content "test file 1" to "textfile.txt" on the server
+        And user "Alice" has set up a client with default settings
+        When the user renames a file "textfile.txt" to "lorem.txt"
+        And the user renames a folder "folder1" to "FOLDER"
+        And the user waits for file "lorem.txt" to be synced
+        And the user waits for folder "FOLDER" to be synced
+        Then as "Alice" file "lorem.txt" should exist on the server
+        And as "Alice" folder "FOLDER" should exist on the server
```
Comment (saw-jan) [inline test/gui/tst_moveFilesFolders/test.feature:55]:
```suggestion
        Then as "Alice" file "lorem.txt" should exist on the server
        And as "Alice" folder "FOLDER" should exist on the server
        But as "Alice" file "textfile.txt" should not exist on the server
        And as "Alice" folder "folder1" should not exist on the server
```

File: test/gui/tst_moveFilesFolders/test.feature [hunk 2]
```diff
@@ -41,4 +41,17 @@ Feature: move file and folder
...
+        And as "Alice" folder "test-folder2" should not exist on the server
+
+
+    Scenario: Rename a file and a folder
+        Given user "Alice" has uploaded file with content "test file 1" to "textfile.txt" on the server
+        And user "Alice" has set up a client with default settings
+        When the user renames a file "textfile.txt" to "lorem.txt"
+        And the user renames a folder "folder1" to "FOLDER"
+        And the user waits for file "lorem.txt" to be synced
+        And the user waits for folder "FOLDER" to be synced
```
Comment (amrita-shrestha) [inline test/gui/tst_moveFilesFolders/test.feature:53]: Do we have a step to wait for syncing all files? I think that was complicated/ impossible to implement so we avoided that. @saw-jan  right?

Comment (amrita-shrestha) [inline test/gui/tst_moveFilesFolders/test.feature:53]: Do we have a step to wait for syncing all files? I think that was complicated/ impossible to implement so we avoided that. @saw-jan  right?

Comment (saw-jan) [reply test/gui/tst_moveFilesFolders/test.feature:53]: Yeah, we have that step.
That step should work well here as well

>  I think that was complicated/ impossible to implement so we avoided that.
Not anymore now. Should be fine here as well

**Review by amrita-shrestha — COMMENTED** [review]
lgtm

**Review by saw-jan — APPROVED** [review]

**Pull request merged**